packages/react-core/src/demos/Toolbar.md

```diff
@@ -0,0 +1,585 @@
+      <DropdownItem key="switch-1">
+        <Checkbox
+          label="Display timestamp"
+          isChecked={firstSwitchChecked}
+          onChange={this.handleFirstSwitchChange}
+          id="switchMobileCheckbox1"
+          name="check1"
+        />
+      </DropdownItem>,
+      <DropdownItem key="switch-2">
+        <Checkbox
+          label="Wrap lines"
+          isChecked={secondSwitchChecked}
+          onChange={this.handleSecondSwitchChange}
+          id="switchMobileCheckbox2"
+          name="check2"
+        />
+      </DropdownItem>,
+      <DropdownSeparator key="separator" />,
+      <DropdownItem key="clear-log" onClick={this.onClearLogs}>Clear logs</DropdownItem>
+    ];
+
+    const selectDropdownContent = (
+      <React.Fragment>
+        {Object.entries(this.firstOptions).map(([value, { type }]) => (
+          <SelectOption
+            key={value}
+            value={value}
+            isSelected={containerSelected === value}
+            isChecked={containerSelected === value}
+          >
+            <Badge key={value}>{type}</Badge>
+            {` ${value}`}
+          </SelectOption>   
+        ))}
+      </React.Fragment>
+    );
+
+    const selectToggleContent = ({ showText }) => {
+      if (!containerSelected) {
+        return "Select";
+      }
+      return (
+        <React.Fragment>
+          <Badge>{this.firstOptions[containerSelected].type}</Badge>
+          {showText && ` ${containerSelected}`}
+        </React.Fragment>
+      );
+    };
+
+    const LogsSearchInput = (
+      <SearchInput 
+        placeholder='find'
+        value={searchValue}
+        onChange={this.onSearchChange}
+        onClear={this.onSearchClear}
+        resultsCount={`${currentSearchResult} / ${searchResultsCount}`}
+        onNextClick={this.onSearchNext}
+        onPreviousClick={this.onSearchPrevious}
+        style={{flex: 1}}
+      />
+    );
+
+    const leftAlignedItemsDesktop = (
+      <React.Fragment>
+        <ToolbarItem visibility={{default: 'hidden', '2xl': 'visible'}}>
+          <Select
+            onToggle={this.onContainerToggle}
+            onSelect={this.onContainerSelect}
+            selections={containerSelected}
+            isOpen={containerExpanded}
+            customContent={selectDropdownContent}
+            placeholderText={selectToggleContent({ showText: true })}
+          />
+        </ToolbarItem>
+        <ToolbarItem visibility={{default: 'hidden', '2xl': 'visible'}}>
+          <Dropdown
+            toggle={
+              <DropdownToggle id='option-toggle-desktop' onToggle={this.onOptionToggle} icon={<CogIcon />}>
+                Options
+              </DropdownToggle>
+            }
+            isOpen={optionExpanded}
+            dropdownItems={optionDropdownItems}
+          />
+        </ToolbarItem>
+        <ToolbarItem visibility={{default: 'hidden', '2xl': 'visible'}}>
+          <Button variant={isPaused? "plain": "link"} onClick={this.pauseOrStart}>
```

Comment:
I like how the mobile tooltips you added to this button - `Pause log` and `Start log` - are more descriptive and likely more accessible than the current "Log Stream" label in the button itself. Maybe the button label should toggle between those two strings when clicked as well? WDYT?

`Pause log` and `Resume log` would actually align well with similar toggles we've started to use elsewhere in OpenShift too:

![image](https://user-images.githubusercontent.com/9122899/102408035-062d5300-3fbb-11eb-99c7-b39672eec613.png)

![image](https://user-images.githubusercontent.com/9122899/102407997-fada2780-3fba-11eb-88cd-1ea92c169746.png)

Reply:
+1. I will change `Log Stream` to `Resume Log` and `Start Log`.